bin/sirius-script-mamba-env-create.bash

```diff
@@ -408,7 +406,7 @@ then
     printf_yellow_clear "Installing accelerators simulation packages.\n"
     clone_or_find lnls lnls-fac && make $TARGET
     clone_or_find trackcpp lnls-fac && make clean && \
-        make install-cpp 2>/dev/null && make $TARGET-py 2>/dev/null
+        make install-cpp 2>/dev/null && make install-py 2>/dev/null
```

Comment:
I think we could remove this redirection of errors to/de/null. What do you think?

Reply:
done